Base/Python/slicer/parameterNodeWrapper/serializers.py

```diff
@@ -192,6 +192,14 @@ def createSerializer(type_, annotations):
     return (ValidatedSerializer(serializer, validators), annotations)
 
 
+def createSerializerFromAnnotatedType(annotatedType):
+    type_, annotations = splitAnnotations(annotatedType)
+    serializer, annotations = createSerializer(type_, annotations)
+    if annotations:
+        print(f"Warning: Unused annotations: {annotations}")
```

Comment:
Probably better to use [`warnings`](https://docs.python.org/3/library/warnings.html) for this.

And the other `print()` warnings, too, for that matter. Probably not a huge deal but the warnings module seems more semantically correct.